reference/attributes.rst

```diff
@@ -27,11 +28,13 @@ Contracts
 Dependency Injection
 ~~~~~~~~~~~~~~~~~~~~
 
+* :class:`Symfony\\Component\\DependencyInjection\\Attribute\\AsAlias`
```

Comment:
would be great to have an example here in a follow up PR, where we can link to

Reply:
Yes definitely! I plan to write some doc in the next days for missing attributes :+1: